LearningGit/LearningGit.cpp

```diff
@@ -1,6 +1,6 @@
 void LearningGitFunction()
 {
     int i = 0;
-    i = i + 1;
+    i = i + 2;
```

Comment:
You know, you could make the increment a function argument, but ok, this change is fine I guess...